packages/react/test/specs/behaviors/alertBehavior-test.tsx

```diff
@@ -0,0 +1,35 @@
+import { alertBehavior } from 'src/lib/accessibility'
+
+describe('AlertBehavior.ts', () => {
+  test('use alertWarningBehavior if warning prop is defined', () => {
+    const property = {
+      warning: true,
+    }
+    const expectedResult = alertBehavior(property)
```

Comment:
```suggestion
    const expectedResult = alertBehavior({ warning: true })
```

nit: Can it be just one-line?